docs/APIRef.LaunchArgs.md

```diff
@@ -0,0 +1,43 @@
+# Launch Arguments
+
+Detox allows in its [`device` API](APIRef.DeviceObjectAPI.md) for the launching the app under test in the test device via an explicit call to `device.launchApp()`. Through various means, Detox enables specifying a set of user-defined arguments (key-value pairs) to be passed on to the app when launched, so as to make them available inside the app itself at runtime (both on the native side, and - if applicable, on the Javascript side).
```

Comment:
Great effort, really, but should I mention that the article is a bit too wordy?
We can revisit it later if we have the capacity, though.

Reply:
Nevertheless, kudos for writing this! 👍